lib/src/utils/movie_dimensions.dart

```diff
@@ -1,11 +1,26 @@
 abstract class MovieDimensions {
   static const double appBarBottomHeight = 45.0;
   static const double appBarTextFieldHeight = 35.0;
-  static const String appBarTextFieldHint = 'Search a movie..';
   static const double appBarTextFieldBorderWidth = 2.0;
   static const double appBarTextFieldContentPadding = 12.0;
   static const bool appBarTextFieldIsFilled = true;
   static const double appBarElevation = 6.0;
   static const int gridBuilderCrossAxisCount = 2;
   static const double gridBuilderMainAxisExtent = 280.0;
+
+  static const num defaultVoteAverage = 0;
+  static const bool defaultIsAdult = false;
```

Comment:
This variable is not necessary here, because you're in Dimensions file

Reply:
Done